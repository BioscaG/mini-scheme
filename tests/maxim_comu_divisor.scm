(define (mcd a b)
  (cond ((= b 0) a)
        (else (mcd b (mod a b)))))

(define (main)
  (display "Introdueix el primer nombre: ")
  (let ((a (read)))
    (display "Introdueix el segon nombre: ")
    (let ((b (read)))
      (display "El màxim comú divisor és: ")
      (display (mcd a b))
      (newline))))
