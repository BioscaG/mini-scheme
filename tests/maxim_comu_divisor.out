Introdueix el primer nombre: Introdueix el segon nombre: El màxim comú divisor és: 2
